docker/Dockerfiledev

```diff
@@ -9,17 +9,13 @@ CMD ["/usr/local/sbin/start.sh"]
 
 FROM ghcr.io/openconext/openconext-containers/openconext-phpfpm-dev:latest AS spdphpfpm
 WORKDIR /var/www/html/
-COPY --from=composer:1 /usr/bin/composer /usr/local/bin/composer
+COPY --from=composer:2.2.9 /usr/bin/composer /usr/local/bin/composer
```

Comment:
Suggestion: remove the version pin. The latest composer will be pulled. This ensures we stay up to date with the latest composer features.

Reply:
The phpcpd gave problems and I tried without versioning and also with :2 and finally it went on with 2.2.9., but finally ended up due to xdebug extension problems. Since phpcpd is no longer used we can use the :latest version I guess.